app/models/api/v1/bookmarks.rb

```diff
@@ -0,0 +1,3 @@
+class Api::V1::Bookmarks < ActiveRecord::Base
```

Comment:
Having looked into this further, I don't think we need this dummy class. The model the bookmark API mass assigns to is the existing `Bookmark` model, so that's the one we need to protect. Adding the `include ActiveModel::ForbiddenAttributesProtection` to model/bookmarks.rb should do the trick (and will then also cover controllers/bookmarks_controller.rb)